Mathlib/FieldTheory/RatFunc.lean

```diff
@@ -583,11 +583,8 @@ def instAddCommGroup : AddCommGroup (RatFunc K) where
 
 instance instCommRing : CommRing (RatFunc K) :=
   { instCommMonoid K, instAddCommGroup K with
-    add := (· + ·)
     zero := 0
```

Comment:
how come you still need the zero and the sub, while you have `instAddCommGroup K` above?

Reply:
This broke and I didn't push it.